interface/patient_file/problem_encounter.php

```diff
@@ -380,6 +381,10 @@ function doclick(pfx, id) {
  }
  if ($alertmsg) echo "alert('" . addslashes($alertmsg) . "');\n";
 ?>
+// reload the parent screen (after saving openemr is not available)
+$('#problem_encounter').submit(function (e) {
+    opener.reloadAfterSecond();
+});
```

Comment:
Wouldn't something like this work(using same mechanism as add_edit_issue.php callback)?
https://github.com/matrix-israel/openemr/blob/cae3ed0675d11226804f3fd9286b1f3aa6390897/interface/patient_file/summary/add_edit_issue.php#L352-L357